webapp/src/main/java/com/box/l10n/mojito/rest/textunit/TextUnitWS.java

```diff
@@ -361,6 +361,7 @@ public TMTextUnitIntegrityCheckResult checkTMTextUnit(@RequestBody TextUnitCheck
             tmTextUnitIntegrityCheckService.checkTMTextUnitIntegrity(textUnitCheckBody.getTmTextUnitId(), textUnitCheckBody.getContent());
             result.setCheckResult(true);
         } catch (IntegrityCheckException e) {
+            logger.info("Integrity check failed for text unit id {}, content {}: {}", textUnitCheckBody.getTmTextUnitId(), textUnitCheckBody.getContent(), e.getMessage());
```

Comment:
Would there be value from also getting the stack trace of the exception logged in this case (vs. just the message of the exception)? (Same question applies to changes in the other file).